site/data/data.json

```diff
@@ -23084,6 +23981,14 @@
         "cashapp": "",
         "last4": "4299"
       },
+      {
+        "hide": true,
+        "name": "Vanessa Morrison",
+        "paypal": "",
+        "venmo": "code?user_id=2714079860883456771",
+        "cashapp": "",
+        "last4": "3881"
+      },
```

Comment:
Hiding due to invalid Venmo name.

Reply:
Venmo sometimes rate-limits the import script, which will try to resolve those URLS. In that case you can try to resolve manually with `venmo.com/<url>`